File: app/src/main/java/com/autoscout24/carfinder/arch/data/network/VehicleWebservice.java
package com.autoscout24.carfinder.arch.data.network;

import com.autoscout24.carfinder.arch.data.database.VehicleEntry;

import retrofit2.Call;
import retrofit2.http.GET;

public interface VehicleWebservice {
    @GET(" ")
    Call<VehicleEntry[]> getAllVehicles();

}

File: app/src/main/java/com/autoscout24/carfinder/arch/data/network/VehicleWebservice.kt
package com.autoscout24.carfinder.arch.data.network

import android.view.View
import com.autoscout24.carfinder.arch.data.database.VehicleEntry
import com.google.gson.annotations.Until
import retrofit2.Call
import retrofit2.http.GET

interface VehicleWebservice {
    @GET(" ")
    fun getAllVehicles(): Call<Array<VehicleEntry>>
}
